benchmark/README.md

```diff
@@ -1,46 +1,15 @@
 # Benchmarks
 
-The following graphs show the results of benchmarking match attempts using different PSI algorithms on various Google Cloud VM's [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard) like n2-standard-64. In each benchmark, the sender and receiver use the same model VM. The receiver has 50m records while the sender has varying datasets of 50m, 100m, 150m, 200m and 250m records. The BPSI used for these experiments has a false positive rate fixed at 1e-6. 
+The following scatter plots show the results of benchmarking match attempts using different PSI algorithms on Google Cloud n2-standard-64 [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard). For each benchmark, the sender and receiver use the same model VM. In the first plot, the receiver has 100m records while the sender has varying datasets of 50m, 100m, 200m, 300m, 400m and 500m records where as in the second plot, the sender's dataset is of 100m records while the receiver has varying datasets of 50m, 100m, 200m, 300m, 400m and 500m records. The BPSI used for these experiments has a false positive rate fixed at 1e-6. 
 
-![n2-standard-32](n2-standard-32.png)
+![scatter_fixed_receiver.png](scatter_fixed_receiver.png)
 
-![n2-standard-48](n2-standard-48.png)
+![scatter_fixed_sender.png](scatter_plot_sender_fixed.png)
 
-![n2-standard-64](n2-standard-64.png)
+The results for match attempts using different PSI algorithms are provided below. Both sender and receiver used n2-standard-64 VMs with datasets containing 50m, 100m, 200m, 300m, 400m and 500m records. The receiver's datasets are represented row-wise while the sender's datasets are represented column-wise.
 
-![n2-standard-80](n2-standard-80.png)
+![bpsi](heatmap_bpsi.png)
```

Comment:
heatmaps look great!
Minor detail, for the x-axis label, you have #Records, but on the scatter plot above, you have #records.
Keep it consistent, or don't include it at all.

Reply:
Unless I am mistaken this looks resolved?

Can we get this merged?  Please also remember to squash the commits.